app/user.go

```diff
@@ -981,6 +981,51 @@ func (a *App) UpdateActive(c request.CTX, user *model.User, active bool) (*model
 		}
 	}
 
+	dmChannels := []*model.Channel{}
+	channelsInTeam, aErr := a.GetChannelsForTeamForUser(c, "", user.Id, &model.ChannelSearchOpts{
+		IncludeDeleted: true,
+		LastDeleteAt:   0,
+	})
+	// soft errors here on since user is updated
```

Comment:
I don't understand this comment.

Reply:
I've updated the comment.

Since the user is already updated, we can't return a server error, thus we just log them.